### DISCUSSION TITLE
huaweicloud_vpc_eip status : "BOUND" or "ACTIVE"

### DISCUSSION THREAD

**Discussion by iabdulrahman91** [huaweicloud/terraform-provider-huaweicloud#1398] [Q&A] [answered]
Hello all,
After creating huaweicloud_vpc_eip using terraform, I noticed the status of huaweicloud_vpc_eip is "ACTIVE" while the status in HuaweiCloud platform show "BOUND". 

Shouldn't be the status described using same word?
Interestingly, when a huaweicloud_vpc_eip get Unbound, the status in Terraform and HuaweiCloud show "UNBOUND".

Thank you

**Comment by niuzhenguo** [comment, marked as answer]
I think we can do this just like we translate "DOWN" to "UNBOUND" on terraform side.

**Reply by iabdulrahman91** [reply]
Yes, this will be really helpful. "BOUND" and "UNBOUND" make way more sense than "ACTIVE" and "DOWN".

**Reply by niuzhenguo** [reply]
you can create a issue for this :D

**Reply by niuzhenguo** [reply]
included in v1.28.0
